toolchain/check/testdata/struct/import.carbon

```diff
@@ -30,9 +30,13 @@ var c: C({.a = 1, .b = 2}) = F();
 // --- fail_bad_type.impl.carbon
 
 impl package Implicit;
-// CHECK:STDERR: fail_bad_type.impl.carbon:[[@LINE+4]]:14: error: missing value for field `a` in struct initialization
+// CHECK:STDERR: fail_bad_type.impl.carbon:[[@LINE+8]]:14: error: missing value for field `a` in struct initialization
 // CHECK:STDERR: var c_bad: C({.c = 1, .d = 2}) = F();
 // CHECK:STDERR:              ^~~~~~~~~~~~~~~~
+// CHECK:STDERR: fail_bad_type.impl.carbon:[[@LINE-4]]:6: in import
```

Comment:
This is out of scope for the current PR, but I wonder if we should omit this message on notes; the phrasing in this context is confusing, and it doesn't seem like it adds anything.

Reply:
Yeah, that might be an improvement.